pages/item/_itemId/index.vue

```diff
@@ -8,7 +8,7 @@
     >
       <div class="d-flex align-end gradient-container">
         <div class="d-flex flex-wrap item-details-container">
-          <div class="itemDetailsLeft">
+          <div class="item-details-left">
```

Comment:
You can use Vuetify utility classes for color: https://vuetifyjs.com/en/styles/colors/